<?xml version="1.0"?>

<launch>
  <node pkg="laser_footprint_filter" type="laser_footprint_filter_node" name="laser_footprint_filter_node" output="screen"> 
      <param name="inscribed_radius_" type="double" value="0.25" /> 
      <param name="scan_sub_topic" type="string" value="scan_rplidar" /> 
      <param name="scan_pub_topic" type="string" value="scan" /> 
      <param name="base_link_name" type="string" value="base_link" /> 
   </node>
</launch>
